README.md

```diff
@@ -1,15 +1,37 @@
-## BATS-core: Bash Automated Testing System (2017)
+# Bats-core: Bash Automated Testing System (2018)
 
 [![Build Status](https://travis-ci.org/bats-core/bats-core.svg?branch=master)](https://travis-ci.org/bats-core/bats-core)
 
-### Background:
+### Table of contents
```

Comment:
This should be a `##` to be at the same level as `## Background`.

Reply:
Silly me...